templates/addons/cilium/cilium-vxlan.yaml

```diff
@@ -0,0 +1,29 @@
+apiVersion: addons.cluster.x-k8s.io/v1alpha1
+kind: HelmChartProxy
+metadata:
+  name: cilium-vxlan
```

Comment:
can we add a `-${CLUSTER_NAME}` to the name of this resource and a selector to only apply to the cluster this is associated with?

Reply:
*slight preference for prefix to follow the convention for the other resources